python/cugraph/dask/link_analysis/mg_pagerank.pxd

```diff
@@ -18,11 +18,12 @@ from cugraph.structure.graph_primtypes cimport *
 from libcpp cimport bool
 
 
-cdef extern from "algorithms.hpp" namespace "cugraph":
+cdef extern from "utilities/cython.hpp" namespace "cugraph::cython":
 
-    cdef void pagerank[VT,ET,WT](
+    cdef void call_pagerank[VT,WT](
```

Comment:
minor: might be a good opportunity to move to the vertex_t, weight_t naming convention for template args

Reply:
updated